miio/huizuo.py

```diff
@@ -0,0 +1,137 @@
+import logging
+import click
+
+from .click_common import command, format_output
+from .device import Device
+from .exceptions import DeviceException
+
+"""
+Basic implementation for HUAYI HUIZUO PISCES For Bedroom (huayi.light.pis123) lamp
+
+This lamp is white color only and supports dimming and control of the temperature from 3000K to 6400K
+Specs: https://miot-spec.org/miot-spec-v2/instance?type=urn:miot-spec-v2:device:light:0000A001:huayi-pis123:1
+
+"""
+
+_LOGGER = logging.getLogger(__name__)
+
+MODEL_HUIZUO_PIS123 = "huayi.light.pis123"
+
+MODELS_SUPPORTED = [MODEL_HUIZUO_PIS123]
+
+
+class HuizuoException(DeviceException):
+    pass
+
+
+class HuizuoStatus:
+    def __init__(self, data):
+        # power = '{"siid":2,"piid":1}' values = true,false
```

Comment:
Could you add the complete response payload (`miiocli -d`) here for future reference?

Reply:
Added in the file as comments (is this what you asked for? ;))

Reply:
Ah, sorry, just the json payload please, here's a good example: https://github.com/rytilahti/python-miio/blob/master/miio/airdehumidifier.py#L59

Reply:
Got it.
Updated by example